wandb/sdk/internal/tb_watcher.py

```diff
@@ -14,6 +15,7 @@
 import wandb
 from wandb import util
 
+from . import file_stream
```

Comment:
lets not include file_stream just for a constant.
Maybe move thee constant elsewhere and have both filestream and tbwatcher include it